tests/components/hassio/test_handler.py

```diff
@@ -92,6 +92,30 @@ async def test_api_host_info_error(hassio_handler, aioclient_mock):
     assert aioclient_mock.call_count == 1
 
 
+async def test_api_core_info(hassio_handler, aioclient_mock):
+    """Test setup with API Home Assistant Core info."""
+    aioclient_mock.get(
+        "http://127.0.0.1/core/info",
+        json={"result": "ok", "data": {"version_latest": "1.0.0"}},
+    )
+
+    data = await hassio_handler.get_core_info()
+    assert aioclient_mock.call_count == 1
+    assert data["version_latest"] == "1.0.0"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)